RSSReader/View/FeedCell.h

```diff
@@ -19,8 +19,13 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, weak) id <CustomTableViewProtocol> delegate;
 @property (nonatomic, assign) int index;
 
+typedef NS_ENUM(NSInteger, cellState) {
```

Comment:
It isn't good naming, please rename it like this (XX your prefix). Prefix is optional, but more preferable with it.
```objc
typedef NS_ENUM(NSInteger, XXCellState) {
     XXCellStateHidden = 0,
     XXCellStateShown = 1
 };
```

Reply:
done